packages/notification-center/src/shared/interfaces/index.ts

```diff
@@ -73,10 +73,10 @@ export interface INovuProviderContext {
 
 export interface INotificationsContext {
   notifications?: Record<string, IMessage[]>;
-  fetchNextPage?: (storeId?: string, query?: IStoreQuery) => void;
+  fetchNextPage?: (storeId?: string) => void;
   hasNextPage?: Record<string, boolean>;
   fetching?: boolean;
-  markAsRead?: (messageId: string, storeId?: string) => Promise<IMessage>;
+  markAsRead?: (messageId: string, storeId?: string) => void;
```

Comment:
Removed the return value from the API because we do not pass in down.
Does some one think that this hook should return the return values from the API?